### PR TITLE
feat: new buyers page

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1541
regen-sanity PR: regen-network/regen-sanity#26

**Update the buyers' page with the following sections:**

- Header (updated)
- 3 items grid (new)
- grid images (updated)
- ecological credit cards section (new)
- featured projects (new)
- partners section (new)
- contact section (updated, previously FAQ)

All sections are wired and customizable through sanity.

**New components:**

In this PR I start adopting the atomic design structure to me more consistent with web-registry components.
- CardRibbon (atom)
- EcologicalCreditCard (molecules)
- Section (organism)
- CarouselSection (organism)

**New fetching hooks:**
- useFetchProjectsByClass
- useFetchProjectsByIds

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1799--regen-registry.netlify.app/buyers

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie here's the link for the new buyers' page: https://deploy-preview-1799--regen-registry.netlify.app/buyers
Note that some links are still missing, and some partners' placeholder icons look odd because I could not export them properly in Figma.

**Comment by erikalogie** [comment]
This looks good!

A few comments:
- Schedule a call should link here: https://calendly.com/sbax_regen/regen-marketplace
- The illustrations look blurry to me, I wonder if they should be exported at 2x so that they look more crisp? (attached)
- You can use the attached logos to replace the ones that look weird when exported from Figma. I am waiting on a full list from Sarah for this.
- Can you make these lines either thinner as they are in the comps or a lighter color so they don't have so much visual weight? https://monosnap.com/file/jLr05LZSUxa7tGUs0xuYlF0f7gjSuQ
- The City Forest credits cards should link to their respective project pages.
- The text "Check out our 2023 credits look book→" should link to the look book in google drive: https://drive.google.com/file/d/1uCu2zkGqkSfNNK3TzjdslFBpcYpx0y-T/view
- The descriptions for Kulshan and ERA Brazil are incorrect
- Will we now be able to change the main top header photo in the CMS?

techstars
stakin

Jungle scene 2
Retirement graphic

**Comment by flagrede** [comment]
@erikalogie here is an update on your comments:

- Schedule a call should link here: https://calendly.com/sbax_regen/regen-marketplace
  - ✅ 
- The illustrations look blurry to me, I wonder if they should be exported at 2x so that they look more crisp? (attached)
  - ✅ 
- You can use the attached logos to replace the ones that look weird when exported from Figma. I am waiting on a full list from Sarah for this.
  - ✅ 
- Can you make these lines either thinner as they are in the comps or a lighter color so they don't have so much visual weight? https://monosnap.com/file/jLr05LZSUxa7tGUs0xuYlF0f7gjSuQ
  - ✅ I used a grey from the theme palette since we can't change the font-weight here because it's a border, let me know if it's ok for you
- The City Forest credits cards should link to their respective project pages.
  - ✅ 
- The text "Check out our 2023 credits look book→" should link to the lookbook in google drive: https://drive.google.com/file/d/1uCu2zkGqkSfNNK3TzjdslFBpcYpx0y-T/view
  - ✅ 
- The descriptions for Kulshan and ERA Brazil are incorrect
  - ✅ took them from the comp initially did not realize it was the same 
- Will we now be able to change the main top header photo in the CMS?
  - ✅ it was not a requirement from the ticket but I just added it

**Comment by erikalogie** [comment]
@flagrede this looks awesome! thanks for all the attention to the little details

Just one last thing that I noticed that this is cut off so ideally the height of the container would expand with the content: https://monosnap.com/file/U7c0ANCb5OL2JeZEQjb3SbBTfwG876

**Review by haveanicedavid — APPROVED** [review]
this story was a lot of work - great job!

File: web-components/src/components/image-grid/imageGrid.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        paddingRight: theme.spacing(10),
+      },
+      [theme.breakpoints.up(theme.breakpoints.values.tablet)]: {
+        flexGrow: 0,
+        flexBasis: '50%',
+        maxWidth: '50%',
+      },
+      [theme.breakpoints.down('sm')]: {
+        paddingRight: theme.spacing(4),
+        paddingLeft: theme.spacing(4),
```
[the remark this answers is not in the document]
Comment (flagrede) [reply web-components/src/components/image-grid/imageGrid.styles.tsx:33]: Actually, I was looking for this syntax this other day, thanks!

**Pull request merged**